CONTRIBUTING.md

```diff
@@ -226,6 +227,42 @@ If you need to make changes to a commit (for example, after receiving comments o
      git push --force
      ```
 
+### Rebasing your branch
+
+Often, when changes are merged into the original repository's `develop` branch, your forked repo may become out of date and you will need to rebase your feature branch. Assuming you added the `mi6/ic-ui-kit` repo as your `upstream` remote branch, you can do the following:
+
+1. Fetch the latest upstream by running:
+```sh
+git fetch upstream
+```
+
+2. Checkout the local `develop` branch by running:
+```sh
+git checkout develop
+```
+
+3. Rebase your local develop off the upstream `develop` branch:
+```sh
+git rebase upstream/develop
+```
+
+4. Checkout your feature branch:
+```sh
+git checkout feature/123-new-feature
+```
+
+5. Rebase your feature branch off the local `develop` branch:
+```sh
+git rebase feature/123-new-feature
+```
+
+6. Push the updated branch:
+```sh
+git push --force

```

Comment:
`git push --force-with-lease`

Reply:
Would this also be required where `git push --force` is mentioned elsewhere in this file (i.e. where it already existed, rather than just added here)?

Reply:
Should other references to force pushing in file also be updated?

Reply:
Yea, but you could do that in a separate PR

Reply:
thanks, updated in this PR